includes/admin/css/geolocator.css

```diff
@@ -1,3 +1,90 @@
+/* Adjust spacing of geolocation metabox a bit */
+#wanderlist-geolocation {
+	margin-top: 15px;
+	margin-bottom: 0px;
+}
+
+#wanderlist-geolocation-map {
+	height: 300px;
+	position: relative;
+	width: 100%;
+}
+
+/* Style "find me" button */
+#wanderlist-locate-user {
+	background-color: #fff;
+	border: 1px solid rgba(0,0,0,.4);
+	border-radius: 3px;
+	box-shadow: none;
+	position: absolute;
+		top: 10px;
+		left: 220px;
+	z-index: 7;
+}
+
+#wanderlist-locate-user::before {
+	color: #404040;
+	content: '\f230';
+	display: block;
+  font-family: dashicons;
+	font-size: 16px;
+	padding: 3px 5px;
+}
+
+#wanderlist-locate-user:hover {
+	background-color: #f8f8f8;
+}
+
+#wanderlist-locate-user:active,
+#wanderlist-locate-user:focus {
+	box-shadow: 0 0 0 1px #5b9dd9,0 0 2px 1px rgba(30,140,190,.8);
+}
+
+/* Spinner for loading */
+.wanderlist-loader {
+	display: none;
+  width: 100px;
+  height: 100px;
+  position: relative;
+  margin: 100px auto;
+	z-index: 10;
+}
+
+.wanderlist-loader.active {
+	display: block;
+}
+
+.wanderlist-loader-circle {
+	animation: wanderlist-bounce 1600ms infinite ease-in-out;
+	background-color: rgba(255, 255, 255, 0.125);
+	border: 2px solid #fff;
+	border-radius: 50%;
+	display: block;
+	height: 100%;
+	position: absolute;
+		top: 0;
+		left: 0;
+  width: 100%;
+}
+
+.wanderlist-loader-circle:nth-of-type(odd) {
+  animation-delay: -800ms;
+}
+
+@keyframes wanderlist-bounce {
```

Comment:
Tested in all browsers? I forget if certain ones need this kind of stuff prefixed.